# CFDE Announcements

## Cross-Pollination Meeting
The next meeting is **Tuesday, October 3rd, at 2 pm ET/ 11 am PT** with presentations scheduled from the **Bridge2AI** program. Regular Cross-Pollination events take place on the first Tuesday of every month. View the [Cross-Pollination Schedule](https://docs.google.com/spreadsheets/d/1hQAeOLkivUZZnwZ_KxfGw3neezMaWbrPk9nnFiKfQGA/edit?usp=sharing) for more details. Email [support@cfde.atlassian.net](mailto:support@cfde.atlassian.net) if you would like to present.

## Four New SenNet-Focused Modules Added to the Visible Human Massive Open Online Course (VHMOOC)
 
We are excited to announce four new modules to the [Visible Human Massive Open Online Course (VHMOOC)](https://expand.iu.edu/browse/sice/cns/courses/hubmap-visible-human-mooc) which now features content covering the Cellular Senescence Network Program (SenNet, https://commonfund.nih.gov/senescence). The VHMOOC aims to:

Communicate tissue data acquisition and analysis
Demonstrate single-cell analysis and CCF mapping techniques
Introduce major features of the HuBMAP portal.
Below, you will find a description on each video and direct links to the [SenNet playlist on YouTube](https://www.youtube.com/playlist?list=PL-CUnYVIy7DPyiFbwCAyveYE8tVUUo25D) under the Cyberinfrastructure for Network Science Center (CNS) channel. 

[Why & How We Construct the Human Reference Atlas](https://iu.instructure.com/courses/1888216/pages/15-why-and-how-we-construct-the-human-reference-atlas)
A tremendous amount of effort and coordination go into constructing, validating, and expanding the Human Reference Atlas (HRA). In this video, IU professor and HuBMAP PI Katy Börner surveys the many moving parts at work in this immense project. She explains how HRA builders meet the challenges of data harmonization and integration and how they develop creative ways of making that data accessible, useful, and engaging. Along the way, Börner introduces us to exciting Atlas features like the Exploration User Interface (EUI), interactive functional tissue units (FTU), 3D organ libraries, and much more.

[SenNet Data Ingest Process](https://iu.instructure.com/courses/1888216/pages/16-sennet-data-ingest-process)
In this useful introduction to the SenNet Data Ingest process, Brendan J. Honick, MLIS, AHIP, a Data Curator & Metadata Librarian at the Pittsburgh Supercomputing Center, takes us step-by-step through how to register samples and datasets, interact with the data sharing portal, and produce visualizations for data analysis. Along the way, he explains why it’s important to understand terms like entity, provenance, protocols, and metadata and how they help us efficiently collect the most accurate medical data.

[Onboarding TMCs: Authoring and Using ASCT+B, and Design of 3D Organ Reference Library Models](https://iu.instructure.com/courses/1888216/pages/17-onboarding-tmcs-authoring-and-using-asct+b-tables-and-designing-3d-organ-reference-library-models)
“Anatomical Structures, Cell Types and Biomarkers (ASCT+B) tables” may be a difficult thing to say but the tables themselves are not difficult to understand and use. Join senior research analyst Ellen M. Quardokus as she explains how these tables help structure knowledge, how they are carefully constructed, and how they can be used the most efficiently. Quardokus then discusses how data from those tables inform the creation of the HRA’s 2D functional tissue unit (FTU) illustrations and 3D reference models.

[Onboarding TMCs: CCF Registration User Interface (RUI) and CCF Exploration User Interface (EUI) ](https://iu.instructure.com/courses/1888216/pages/18-onboarding-tmcs-ccf-registration-user-interface-rui-and-exploration-user-interface-eui)
Indiana University research scientist Andreas Bueckle provides a clear and concise demonstration of how researchers can use the RUI to spatially register 3D tissue blocks into reference organs and how they can employ the EUI to explore the data they and their colleagues generate. As a bonus, Bueckle gives us an exclusive peek at an in-the-works spatial search feature that allows users to collect information by moving in and out of 3D reference organs.




## Kids First ASHG Poster Session - An Invitation
The NIH [Gabriella Miller Kids First Pediatric Research Program (Kids First)](https://kidsfirstdrc.org/) and [INvestigation of Co-occurring conditions across the Lifespan to Understand Down syndromE (INCLUDE)](https://includedcc.org/) are hosting an ancillary poster session at the 2023 American Society of Human Genetics (ASHG) Annual Meeting in Washington, DC on  **Wednesday, November 3 at 7:15 - 9:15 p.m. ET**. The theme of this meeting is “Accelerating Pediatric Genomics Research Through Collaboration.” Presentations will feature information about whole genome datasets from both programs, progress of Kids First and INCLUDE cohort projects, tools and functionalities of the Kids First Data Resource Portal and the INCLUDE Data Hub, and collaborations that can amplify the impact of Kids First and INCLUDE research efforts. Kids First is inviting all researchers whose work involves collaborations with or data use from Kids First and/or INCLUDE to present a poster with an overview of the research and, if you would like, information about how the Kids First and/or INCLUDE research communities can collaborate with you as well. You are welcome to reuse posters you have created in the past, a variety of poster sizes can be accommodated. If you have submitted a poster to the ASHG meeting, you are welcome to use it for this purpose as well.  

## CFDE Funding Opportunities
**Notice of Funding Opportunity ([RFA-RM-23-013](https://grants.nih.gov/grants/guide/rfa-files/RFA-RM-23-013.html))**, which seeks proposals for projects that involve collaboration with multiple Common Fund Data Coordinating Centers (DCCs) to combine discovery and/or translational approaches with work that will enhance reusability of Common Fund data and contribute knowledge to the Common Fund Data Ecosystem. Proposed projects should improve the overall reuse potential of Common Fund data by developing and testing services needed to efficiently access, analyze, and, if necessary, integrate the data. Applications must involve a collaboration between at least two Common Fund DCCs listed in the Background section of the Notice and at least one external PI not currently funded by a CF program. 

Letters of intent are required for proposals requesting $500,000 or more in direct costs in any one budget year (excluding consortium and F&A costs), and are due **November 4, 2023** by 5:00 PM local time of the applicant organization. Full proposals are due **December 4, 2023**, by 5:00 PM local time of the applicant organization.

 **Notice of Funding Opportunity ([RFA-RM-23-015](https://grants.nih.gov/grants/guide/rfa-files/RFA-RM-23-015.html))**, Pilot Projects Enhancing Utility and Usage of Common Fund Data Sets (R03 Clinical Trial Not Allowed) which seeks pilot projects that will demonstrate and enhance the utility of selected Common Fund data sets, including generating hypotheses and catalyzing discoveries. Investigators are encouraged to utilize various approaches including, but not limited to, systems approaches, artificial intelligence (including generative)/machine learning/deep learning methods, advanced data science methods for data set integration and harmonization, and incorporating computational modeling to bring together high-throughput genotype and phenotype data sets. Proposed research plans must substantially leverage data from at least two eligible Common Fund programs detailed in the announcement.  

Full proposals are due  **November 17, 2023,  by 5:00 PM** local time of the applicant organization.   

Learn more on the [Common Fund Data Ecosystem](https://commonfund.nih.gov/dataecosystem) website and contact the CFDE team at [CFDE@od.nih.gov](mailto:CFDE@od.nih.gov) with any additional questions.    

## Working Group Updates
You can find a list of CFDE WGs and linked agendas on the [Cross-Pollination Meeting Schedule](https://docs.google.com/spreadsheets/d/1hQAeOLkivUZZnwZ_KxfGw3neezMaWbrPk9nnFiKfQGA/edit?usp=sharing). Currently scheduled working group meetings are as follow: 
| Working Group | Date | Time | Agenda |
| ----------------- | ----- | ----- | --------- | 
Knowledge Graph |September 19th | 12pm PT / 3pm ET | [Knowledge Graph WG](https://docs.google.com/document/d/1WvpkLxWPW0XxZsam6jEJeEUQr2sQ0EWC/edit?usp=sharing&ouid=111367545760360703840&rtpof=true&sd=true)
Ontology | September 20th  | 8am PT / 11am ET | [Ontology WG](https://docs.google.com/document/d/1VoHHBeWfol6XNJa3kzOnOFuTaIrcLYbqKYQcOnj1oh4/edit?usp=sharing)

## C2M2 Submission Schedule
| Submission Deadline | Portal Update |
| ---------------------- | -----------------|
Sept 15 | Oct 1
Dec 15 | Jan 1
Mar 15 | Apr 1
 
## Welcome
**GlyGen**: Sean Kim

## Contributing
You can contribute to the next announcements by commenting on the open announcements issue in [this repository](https://github.com/nih-cfde/announcements/issues). Share your events and updates by posting links and descriptions in our announcements issue! If you have questions about these or other events/activities in the CFDE, contact our helpdesk at [support@cfde.atlassian.net](mailto:support@cfde.atlassian.net).

Thank you all for your efforts and have a great week!
