# CFDE Announcements -- DRAFT

## Cross-Pollination Events
Our next scheduled meeting will take place on **Tuesday, January 4th at 11am PT / 2pm ET**. January's meeting is scheduled to have a second round of presentations from R03 awardees detailing their progress and experiences using CF data and tools.  Regular Cross-Pollination events take place on the first Tuesday of every month. Please check out the  [Cross-Pollination Schedule](https://docs.google.com/spreadsheets/d/1hQAeOLkivUZZnwZ_KxfGw3neezMaWbrPk9nnFiKfQGA/edit?usp=sharing) for more details.

## CFDE Training Efforts
Our two part Cloud Computing series continues this week with our popular [Intro to AWS ](https://www.nih-cfde.org/events/introduction-to-amazon-web-services-2/?pk_campaign=anc)workshop on **December 15th**. Check out our [Events Calendar](https://www.nih-cfde.org/events/) for details and registration availability.

## RFCs Open for Comment
Two RFC’s are now available for comment by the community until January 11th.  
•	[Considerations about future gene pages and gene-centric dashboard for CFDE](https://docs.google.com/document/d/1n9sKKGcXpZscG-eGClg7RmUs8fzYbeQ1/edit)
•	[Variant Information and Knowledge Exchange Principles]( https://docs.google.com/document/d/1bM2L3Ux3CrGISUVETVeVcvK8e0XT_Zey/edit) 
RFC stands for “Requests for Comments”. These are proposals being made by members of the CFDE to suggest that the CFDE community as a whole adopt the suggested new best practice. Everyone in the community is invited to respond to these proposals by leaving comments, critiques, and support at the links above. Comments are open for one month at which point the documents will be reviewed. If there are no substantial objections to the RFC in that time, it will be adopted as a CFDE Best Practice. You can read more about our RFC process here.

The [Use of Uberon for anatomy in the C2M2](https://docs.google.com/document/d/1x-KYUOvzO6NMiHqnl9G8JzakLcqJbkfG/edit)  RFC is also still available for comment by the community until December 15th. 


## IDG Tools
The IDG Team has recently produced several video presentations we would like to share:
https://youtu.be/a0tbojE9DEU In this e-IDG presentation, Anna Calinawan, a Software Engineer from the Wang Lab, describes a collaborative project between the Ma’ayan Lab and Pei Wang Lab at ISMMS where we apply kinase enrichment analysis to analyze phosphoproteomics data from the NCI CPTAC program. The web portal that Anna developed can be accessed at: https://pancan-kea3.cptac-data-view.org.

https://youtu.be/6gDzA2dfOrI In this e-IDG lecture Dr. Alexander Lachmann, an Assistant Professor from the Ma’ayan Lab at ISMMS, describes three projects: PrismExp, blitzGSEA, and bridgeGSEA. These are three related methods to predict gene function and incorporate such predictions within gene set enrichment analyses. Read more about PrismExp here: https://www.biorxiv.org/content/10.11.... The PrismExp Appyter can be accessed from here:https://appyters.maayanlab.cloud/#/Pr...

https://youtu.be/m2bVYhF7MHo In this e-IDG lecture, Eryk Kropiwnicki, a Data Scientist from the Ma’ayan Lab, presents DrugShot, a web-based application and an Appyter that converts PubMed searches into ranked lists of drugs and other small molecules based on literature co-mentions. DrugShot also makes predictions about additional relevant drugs based on drug-drug LINCS L1000 gene expression similarity. The DrugShot website can be accessed at: https://maayanlab.cloud/drugshot/. The DrugShot Appyter can be accessed at:https://appyters.maayanlab.cloud/#/Dr....

The IDG Digital Tool Fest 2021 was a free virtual meeting with demonstrations of different IDG digital tools developed for analyzing, visualizing and interrogating target-centric and drug-centric information.  The meeting took place on November 30th, 2021. You can view recordings on YouTube: https://youtube.com/playlist?list=PL-8k9c6dRYowAMrI0L812G6TNmZTcBDWv

## Working Group Updates

### Gene Expression Data Exchange Working Group
This new working group is kicking off **Monday, January 3rd at 11am PT / 2pm ET**.  This group will survey and evaluate existing resources and then recommend standards for metadata, file formats, and an API for the CFDE to facilitate storing, retrieving, and querying gene expression data. For more detail see the [WG charter](https://docs.google.com/document/d/1D_Kjn_yBIYibco33zsmfBMrC0C39IGwW_VPu9b_iXnM/edit).  Join our mailing list by sending a message to [GeneExpressionDataExchange+subscribe@CFDEpublic.groups.io](GeneExpressionDataExchange+subscribe@CFDEpublic.groups.io) .

### Gene Working Group
The next meeting for the Gene Working Group is scheduled for **Wednesday, January 12th at 11am PST / 2pm EST**. The Zoom link for the meeting can be found at the top of the [rolling agenda](https://docs.google.com/document/d/18QXDCFkHTVF2LTvab-wz9CprHxegP6VU/edit#). Please join our [mailing list](https://cfdepublic.groups.io/g/GeneWorkingGroup) and Slack channel (gene-wg) for more details about working group activities.

### Ontology Working Group
The next meeting for the Ontology Working Group is scheduled for **Wednesday, December 15th at 8am PT / 11am ET**. Ontology Working Group meetings are scheduled bi-weekly on Wednesdays. RFC and Survey 'homework' tasks can be found in the [rolling agenda](https://docs.google.com/document/d/1VoHHBeWfol6XNJa3kzOnOFuTaIrcLYbqKYQcOnj1oh4/edit#heading=h.3ia46913z0oa). You can get involved by joining their [mailing list](https://cfdepublic.groups.io/g/OntologyWorkingGroup) and [slack channel](https://join.slack.com/share/zt-wramurmc-0VP3wp~RYL8y1VPndQvYXw). 

### Variant Working Group
This new working group is now holding meetings on the second Monday each month. The next meeting will take place on **Monday, January 10th at 1pm PT / 4pm ET**. The [charter](https://docs.google.com/document/d/1L84L20Z3v4wPYjdqjQz0JpGFud0JPNUK/edit?usp=sharing&ouid=111367545760360703840&rtpof=true&sd=true) and [agenda](https://docs.google.com/document/d/1c3bxCKCRTWtvZopSLOT2iZsetylKtqdilfF1hB1thFQ/edit?usp=sharing) are available for viewing. Join the conversation on our [slack channel](https://join.slack.com/share/zt-wr77bkhw-RrrNoi4OaOgG~u7D2DFayA)!

## Contributing
You can contribute to the next announcements by commenting on the open announcements issue in [this repository](https://github.com/nih-cfde/announcements/issues). Share your events and updates by posting links and descriptions in our announcements issue!

Thank you all for your efforts and have a great week!

Jeremy Walter
